pom.xml

```diff
@@ -115,7 +115,8 @@
     <module>google-cloud-pubsub</module>
     <module>google-cloud-resourcemanager</module>
     <module>google-cloud-spanner</module>
-    <module>google-cloud-speech</module>
+    <module>google-cloud-speech-v1</module>
+    <module>google-cloud-speech-v1beta1</module>
```

Comment:
Do we have any existing clients that contain more than one version? If we merge two versions into one, they would share a single pom. In the last discussion of releasing Speech, the plan is to have both versions sit side by side independently.